### ISSUE TITLE
[WINDOW ISSUE] so when i use samsung dex the gray bar up the top messes minecraft buttons up

### ISSUE THREAD

**Issue by mario6413** [PojavLauncherTeam/PojavLauncher#2413]
### Describe the bug

so when i load minecraft it runs fine but the buttons are off centured so when my mouse is on the singleplayer it would highlight multiplayer i dont know if you can fix the issue for samsung dex but it would be very helpful

### Steps To Reproduce

```markdown
1. start pojav
2. minecraft loads
```

### Expected Behavior

i expect minecraft to change res for samsung dex

### Platform

```markdown
- Device model: Galaxy S20FE
- CPU architecture: aarch64
- Android version: 11
- PojavLauncher version: crocus-v3_openjdk
```

**Comment by ShadowFox1019** [comment]
Hey, I made a bug report about this around 6 months ago, and they never really resolved it. After a while I realized that—for some reason that escapes my comprehension—when I open pojav, as it's loading up on the red mojang screen, if I open something like
Samsung internet in a full screen tab above pojav then count to about 5 or 10 and then minimize the tab, the bar dissapears as pojav is displayed again. The buttons are then correctly positioned for me. I have no idea why this weird glitch works and if it's specific to my device and versions (etc.), but maybe it's worth giving it a try, especially since I brought it up to a dev and spoke to them about half a year ago, and nothing has changed.
